client/rest-high-level/src/main/java/org/elasticsearch/client/MachineLearningClient.java

```diff
@@ -166,4 +168,42 @@ public void openJobAsync(OpenJobRequest request, RequestOptions options, ActionL
             listener,
             Collections.emptySet());
     }
+
+    /**
+     * Closes Machine Learning Jobs
```

Comment:
nit: Merge this into the next sentence? Something like `Closes one or more Machine Learning jobs...`